lib/stat_tracker.rb

```diff
@@ -513,77 +513,71 @@ def worst_coach(season)
     # games.csv - game_id, season 
     # teams.csv - team_id, teamName
 
-  def most_accurate_team
+  def most_accurate_team(season)
```

Comment:
I do think it's extra rad you managed to get this method functioning while evaluating ALL seasons, just saying